pkg/volume/glusterfs/glusterfs.go

```diff
@@ -490,13 +495,15 @@ func (r *glusterfsVolumeProvisioner) Provision() (*api.PersistentVolume, error)
 	if len(pv.Spec.AccessModes) == 0 {
 		pv.Spec.AccessModes = r.plugin.GetAccessModes()
 	}
+	sGid := strconv.FormatInt(reqGid, 10)
+	pv.Annotations = map[string]string{"pv.beta.kubernetes.io/gid": sGid}
```

Comment:
@erinboyd yes, this is a couple months old.  this will make it so that the pod that consumes the claim will run with the supplemental group in the annotation . Without there needing to be communication between the claimant and the PV creator (I.e. So that the claimant does not need to fill in the supp. Group field on the pod)

The issue @humblec is talking about is, I think, that the use of this annotation bypasses any access/pap restrictions on a pod's supplemental group